packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -132,13 +131,13 @@ export class Omnibox<T> extends AbstractComponent<IOmniboxProps<T>, IOmniboxStat
                 hasBackdrop={true}
                 {...overlayProps}
                 isOpen={isOpen}
-                className={classNames(overlayProps.className, `${Classes.OMNIBOX}-overlay`)}
+                className={classNames(overlayProps.className, Classes.OMNIBOX_OVERLAY)}
                 onClose={this.handleOverlayClose}
             >
                 <div
                     className={classNames(listProps.className, Classes.OMNIBOX)}
-                    onKeyDown={isOpen && query.length > 0 && handleKeyDown}
-                    onKeyUp={isOpen && query.length > 0 && handleKeyUp}
+                    onKeyDown={isOpen && query.length > 0 ? handleKeyDown : undefined}
+                    onKeyUp={isOpen && query.length > 0 ? handleKeyUp : undefined}
```

Comment:
another approach, since both cases are the same:

```js
const handlers = isOpen && query.length > 0
    ? { onKeyDown: handleKeyDown, onKeyUp: handleKeyUp }
    : {};

<div props {...handlers}>
```